### PR TITLE
Clean up some SQL code and use string instead of BLOB

### PR DESCRIPTION
**Pull request by Geod24**
See each commit.

### CODE REVIEW

File: source/agora/consensus/EnrollmentManager.d [hunk 1]
```diff
@@ -161,7 +160,7 @@ public class EnrollmentManager
 
         // create the table for enrollment data for a node itself
         this.db.execute("CREATE TABLE IF NOT EXISTS node_enroll_data " ~
-            "(key CHAR(128) PRIMARY KEY, val BLOB NOT NULL)");
+            "(key CHAR(128) PRIMARY KEY, val CHAR(66) NOT NULL)");
```
Comment (TrustHenry) [inline source/agora/consensus/EnrollmentManager.d]:
```suggestion
            "(key CHAR(128) PRIMARY KEY, val TEXT(66) NOT NULL)");
```

Comment (Geod24) [reply source/agora/consensus/EnrollmentManager.d]: Actually, according to SQLite manual, it just ignore the length, so I'm going to make it `TEXT`

**Comment by Geod24** [comment]
Updated, thanks for spotting this @TrustHenry !

**Review by TrustHenry — APPROVED** [review]
LGTM

**Pull request merged**